src/AvaloniaEdit/Utils/TextFormatterFactory.cs

```diff
@@ -27,7 +27,7 @@ namespace AvaloniaEdit.Utils
     /// <summary>
     /// Creates TextFormatter instances that with the correct TextFormattingMode, if running on .NET 4.0.
     /// </summary>
-    internal static class TextFormatterFactory
+    public static class TextFormatterFactory
```

Comment:
Why was this made public? It was internal in AvalonEdit. Besides, it doesn't do much, I was thinking of removing it altogether. You can just use `new TextFormatter()`.